paddle/fluid/imperative/bkcl_context.h

```diff
@@ -0,0 +1,53 @@
+//   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+
+#if defined(PADDLE_WITH_XPU_BKCL)
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "paddle/fluid/imperative/parallel_context.h"
+#include "xpu/bkcl.h"
+
+namespace paddle {
+namespace imperative {
+
+class BKCLParallelContext : public ParallelContext {
```

Comment:
maybe we can add some notes for `BKCL` here

Reply:
Done